.taskcluster.yml

```diff
@@ -177,6 +177,35 @@ tasks:
           owner: ${user}@users.noreply.github.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("update_hook")}
```

Comment:
Before merging, we should only run this task on tags, and only after the deployment of the docker images is done (so you can add the docker push task as a dependency of this task).

Reply:
Do we want also to move to taskboot new command before merging?

Reply:
I think it's basically ready, right? If so, yes.

Reply:
Note: to deploy only on tags, you'll need to add an if like the one we have for the PyPI release task.

Reply:
Got it, I'm waiting for the taskboot PR to be merged and to have validated the dependencies before squashing and adding the condition